Database/Corrections/sodQuestFixes.lua

```diff
@@ -100,6 +100,12 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.zoneOrSort] = sortKeys.WARLOCK,
             [questKeys.requiredSpell] = -403919,
         },
+        [77669] = {
+            [questKeys.objectives] = {nil, nil, nil, nil, nil, {{410002, nil, 205420}}},
+            [questKeys.requiredRaces] = raceIDs.UNDEAD,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+            [questKeys.requiredSpell] = -410002,
```

Comment:
```suggestion
            [questKeys.requiredSpell] = -400105,
```

This needs to be the [Engrave](https://www.wowhead.com/classic/spell=400105/engrave-gloves-shadowstrike) version of the spell.